frontend/mr-admin-flate/src/mocks/endpoints/tiltaksgjennomforingNotatHandlers.ts

```diff
@@ -0,0 +1,65 @@
+import { rest } from "msw";
+import {
+  TiltaksgjennomforingNotat,
+  TiltaksgjennomforingNotatRequest,
+} from "mulighetsrommet-api-client";
+import { mockTiltaksgjennomforingnotater } from "../fixtures/mock_tiltaksgjennomforingnotater";
+
+let tiltaksgjennomforingNotater = [...mockTiltaksgjennomforingnotater];
+
+export const tiltaksgjennomforingNotatHandlers = [
+  rest.get<any, any, TiltaksgjennomforingNotat[]>(
+    "*/api/v1/internal/notater/tiltaksgjennomforinger",
+    (req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json(tiltaksgjennomforingNotater.sort(sortByDate)),
+      );
+    },
+  ),
+  rest.get<any, any, TiltaksgjennomforingNotat[]>(
+    "*/api/v1/internal/notater/tiltaksgjennomforinger/mine",
+    (req, res, ctx) => {
+      return res(
+        ctx.status(200),
+        ctx.json(
+          tiltaksgjennomforingNotater
+            .filter((notat) => notat.opprettetAv.navIdent === "B99876")
+            .sort(sortByDate),
+        ),
+      );
+    },
+  ),
+
+  rest.put<TiltaksgjennomforingNotatRequest, any, any>(
+    "*/api/v1/internal/notater/tiltaksgjennomforinger",
+    async (req, res, ctx) => {
+      const payload = await req.json<TiltaksgjennomforingNotatRequest>();
+      tiltaksgjennomforingNotater.push({
+        ...payload,
+        createdAt: new Date().toISOString(),
+        updatedAt: new Date().toISOString(),
+        opprettetAv: { navIdent: "B99876", navn: "Bertil Betabruker" },
+      });
+      return res(ctx.status(200));
+    },
+  ),
+
+  rest.delete<any, any, any>(
+    "*/api/v1/internal/notater/tiltaksgjennomforinger/:id",
+    (req, res, ctx) => {
+      const id = req.url.searchParams.get("id");
```

Comment:
Endre denne til `const {id} = req.params;` for å slippe samme bug som med avtalenotater